Packs/OpenAI/Integrations/OpenAI/README.md

```diff
@@ -1,63 +1,238 @@
+| entry_id | Entry ID of file for the model to summarize. Either arg "text" or "entry_id" is required. | Optional | 
+| prompt | Additional prompt to optionally add more specific instructions. | Optional | 
+| model | Name of model to use. Used for Azure OpenAI only. Default is text-davinci-003. | Optional | 
+| deployment | Name of deployment to use. Used for Azure OpenAI only. Default is text-davinci-003. | Optional | 
+| temperature | Controls randomness: Lowering results in less random completions. Default is 0. | Optional | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| OpenAI.Summary | unknown | Summary returned by the model | 
+
+#### Command example
+```!openai-summarize entry_id=1504@3d4f3713-0634-41eb-833b-299c6253eea9```
+#### Context Example
+```json
+{
+    "OpenAI": {
+        "Summary": "This guide provides best practices for optimizing the performance of Cortex XSOAR, a security orchestration, automation, and response platform. It includes configuration best practices, such as enabling signature verification on incoming SAML authentication tokens, restricting workers for all time searches, and database optimizations. Additionally, it provides help on playbook design and best practices, such as using Quiet Mode and disabling auto-extract on tasks that do not require it. It also provides instructions on how to configure a server to restrict the number of workers allowed to execute parallel searches, as well as how to audit index tuning and purge existing items. Additionally, it provides guidance on how to administer, tune, monitor, and maintain Cortex XSOAR for multi-tenant hosts and HA groups."
+    }
+}
+```
+
+#### Human Readable Output
+
+>**Summary**: This guide provides best practices for optimizing the performance of Cortex XSOAR, a security orchestration, automation, and response platform. It includes configuration best practices, such as enabling signature verification on incoming SAML authentication tokens, restricting workers for all time searches, and database optimizations. Additionally, it provides help on playbook design and best practices, such as using Quiet Mode and disabling auto-extract on tasks that do not require it. It also provides instructions on how to configure a server to restrict the number of workers allowed to execute parallel searches, as well as how to audit index tuning and purge existing items. Additionally, it provides guidance on how to administer, tune, monitor, and maintain Cortex XSOAR for multi-tenant hosts and HA groups.
+
+## Breaking changes from the previous version of this integration - OpenAI
+The following sections list the changes in this version.
+
+### Commands
+#### The following commands were removed in this version:
+* *chatgpt-send-prompt* - this command was replaced by *openai-chatgpt*.
+
+## Additional Considerations for this version
```

Comment:
Maybe also add that the `OpenAIChatGPTV3` integration has been combined with the `OpenAI` integration, and the command *chatgpt-send-prompt`* is now the *openai-chatgpt* commmand.